src/index.html

```diff
@@ -23,13 +23,30 @@ <h2>Recent Scores</h2>
       <div class="table" id="score-input">
         <div class="title">
           <h2>Add Your Score</h2>
+          <button id="toggle-mute" type="button" class="btn"><i class="fa-solid fa-music"></i></button>
         </div>
         <div id="inputs">
           <input id="user" type="text" placeholder="Name" required>
           <input id="score" type="text" placeholder="Score" required>
```

Comment:
- [ ] [Optional] [Highly Recommended] The score input field suppose to submit numbers only So kindly change its type to solve this problem.

### ⚠️I marked this as optional because this is a requirement in older milestone for this project, so please consider doing it⚠️